modules/server/src/main/ts/bedrock/auto/RemoteDriver.ts

```diff
@@ -70,14 +70,29 @@ export const getApi = async (settings: DriverSettings, browser: string, opts: We
     const driver = await WebdriverIO.remote(opts);
     return {
       webdriver: driver,
-      shutdown: () => Promise.resolve()
+      shutdown: () => driver.deleteSession()
     };
   }
   return Promise.reject('Unrecognized remote provider: [' + remoteWebdriver + ']');
 };
 
 const addDriverSpecificOpts = (opts: WebdriverIO.RemoteOptions, settings: DriverSettings): WebdriverIO.RemoteOptions => {
   if (settings.remoteWebdriver === 'lambdatest') {
+    // For naming in LT we use PROJECT_BUILD[_NAME] or BUILD
```

Comment:
Do we enforce this format in the settings so that we don't pass in some format that lambda test doesn't support? And is that format documented somewhere can we point to it.

Reply:
LT allows user to specify a `build`, `project` and test `name` as part of the options map. There is no specific format. I didn't want to keep adding Client Options to bedrock so my idea was to get that from the name. If no name is passed it defaults to `bedrock-auto` for the name of the build. We can either add the extra effort of accepting client options but it only will be used for LT or we can identify one string pattern to pass via the name